docs/guides/migrate-to-zeebe-user-tasks.md

```diff
@@ -0,0 +1,376 @@
+
+:::caution
+Make sure to migrate from embedded to linked forms first before changing the implementation type. Use the undo function in case you accidently removed your embedded form.
+:::
+
+## Use the new Zeebe Task API
+
+:::note
+The Tasklist REST API is not deprecated, you still need it for queries on both task types.
+:::
+
+Operations on Zeebe user tasks which modify the task state have to be performed using the new Zeebe REST API. However, queries and adjacent operations still require the Tasklist REST API. The following table provides a breakdown of which operations are supported in which API, and for which user tasks.
+
+<table style={{ textAlign: "center" }}>
+    <tr>
+        <th style={{ textAlign: "end" }}>Operation</th>
+        <th>Tasklist API</th>
+        <th>Zeebe Task API (8.5)</th>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Query tasks</th>
+        <td><span style={{ color: "green" }}>✔</span> All types</td>
+        <td style={{color: "gray"}}>← Use Tasklist API</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Get task</th>
+        <td><span style={{ color: "green" }}>✔</span> All types</td>
+        <td style={{color: "gray"}}>← Use Tasklist API</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Retrieve task variables</th>
+        <td><span style={{ color: "green" }}>✔</span> All types</td>
+        <td style={{color: "gray"}}>← Use Tasklist API</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Get task form</th>
+        <td><span style={{ color: "green" }}>✔</span> All types</td>
+        <td style={{color: "gray"}}>← Use Tasklist API</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Change task assignment</th>
+        <td><span style={{ color: "green" }}>✔</span> Job-based tasks</td>
+        <td><span style={{ color: "green" }}>✔</span> Zeebe tasks</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Complete task</th>
+        <td><span style={{ color: "green" }}>✔</span> Job-based tasks</td>
+        <td><span style={{ color: "green" }}>✔</span> Zeebe tasks</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Update task</th>
+        <td>-</td>
+        <td><span style={{ color: "green" }}>✔</span> Zeebe tasks</td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Safe + retrieve draft variables</th>
+        <td><span style={{ color: "green" }}>✔</span> All types</td>
+        <td style={{color: "gray"}}>← Use Tasklist API</td>
+    </tr>
+</table>
+
+You can also operate both task types at the same time in the same application by utilizing both APIs. We recommend this for a smooth migration, but you should eventually update all processes to use the new task type to use all benefits. The following image illustrates how to route API calls to the respective APIs:
+
+<img src={APIArchitectureImg} className={styles.noShadow} style={{width: 800}} alt="Task API Architecture" />
```

Comment:
Blocking: 

The images on this page have the shadow scrubbed, like the image below. [We previously found that images without any border or shadow were confusing](https://github.com/camunda/developer-experience/issues/24), so we added a shadow. @christian-konrad, you opened #2059 to remove the _shadow_ (but not the border); we have not moved on that issue, because the border vs outline debate seems to me to be subjective opinion. 

I would be fine with these images removing the _shadow_ without also removing the outline, but removing both is a regression in terms of readability, supported by the feedback we originally received for images with no outline. 

cc @christinaausley and @akeller because I think they probably also have opinions about the presence of borders on images. 

<img width="865" alt="image" src="https://github.com/camunda/camunda-docs/assets/1627089/c278db46-9689-457d-9365-8b22c0cc0786">

Reply:
I can readd the border; the drop-shadow just decreases readability in my opinion and elevates the images were it's not needed, but I agree to the border.